libs/src/services/creatorNode/index.js

```diff
@@ -543,7 +571,21 @@ class CreatorNode {
 
     // Axios throws for non-200 responses
     try {
+      const start = Date.now()
       const resp = await axios(axiosRequestObj)
+      const duration = Date.now() - start
+
+      if (this.monitoringCallbacks.request) {
```

Comment:
`if (this.monitoringCallbacks && this. monitoringCallbacks.request)` ? in case `monitoringCallbacks` is null

Reply:
+1, ill make the default {} so this check is shorter :)